kura/org.eclipse.kura.ai.triton.server/OSGI-INF/metatype/org.eclipse.kura.ai.triton.server.TritonServerContainerService.xml

```diff
@@ -158,6 +158,33 @@
             default="4194304"
             min="1">
         </AD>
+        
+        <AD id="enable.metrics"
+            name="Enable Triton Server Metrics"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="true"
+            description="Enable the Triton Server Metrics feature. This property enables the default, CPU and GPU metrics, if available.">
+        </AD>  
+        
+        <AD id="metrics.config"
+            name="Optional metrics-specific configuration"
+            type="String"
+            cardinality="0"
+            required="false"
+            default=""
+            description="A semi-colon separated list of metrics-specific configuration settings for the Triton Server Metrics. (i.e. counter_latencies=true)">
```

Comment:
```suggestion
            description="A semi-colon separated list of Triton Server Metrics. (i.e. counter_latencies=true)">
```

Reply:
@MMaiero This property allows to configure a metric, enabling or disabling the statistics that are emitted by the service. For example, when the metrics are enabled, a user can disable the `counter_latencies` setting `--metrics-config counter_latencies=false`. So, they aren't new metrics, but configurations for metrics that are already there.